YahooFinanceApi/Candle.cs

```diff
@@ -4,29 +4,18 @@ namespace YahooFinanceApi
 {
     public class Candle: ITick
     {
-        public Candle(DateTime dateTime, decimal open, decimal high, decimal low, decimal close, long volume, decimal adjustedClose)
-        {
-            DateTime = dateTime;
-            Open = open;
-            High = high;
-            Low = low;
-            Close = close;
-            Volume = volume;
-            AdjustedClose = adjustedClose;
-        }
+        public DateTime DateTime { get; internal set; }

```

Comment:
Removing this constructor and changing the properties to have internal set means this object is no longer immutable. Which IMO is a bad idea as something else in the library can now mess with the values after they have been set.

Reply:
Thanks for looking at the code and providing feedback.

Yes, using internal setters rather than a constructor makes the objects mutable from within the library. But in this case there is little benefit in making these objects immutable since they are just simply sent to the caller. Using setters allows for initialization code that is shorter, more intuitive and less error-prone. I believe this results in better code.

Reply:
Yeh fair point